reconcile/dynatrace_token_provider/integration.py

```diff
@@ -69,7 +71,30 @@ def run(self, dry_run: bool) -> None:
         dependencies.populate()
         self.reconcile(dry_run=dry_run, dependencies=dependencies)
 
+    def _token_cnt(self, dt_tenant_id: str, ocm_env_name: str) -> None:
+        with self._lock:
+            if ocm_env_name not in self._managed_tokens_cnt:
+                self._managed_tokens_cnt[ocm_env_name] = {}
+            if dt_tenant_id not in self._managed_tokens_cnt[ocm_env_name]:
+                self._managed_tokens_cnt[ocm_env_name][dt_tenant_id] = 0
+            self._managed_tokens_cnt[ocm_env_name][dt_tenant_id] += 1
```

Comment:
by using `defaultdict` and `Counter`, all key checks can be saved

first init `_managed_tokens_cnt` as `defaultdict`

```python
    self._managed_tokens_cnt = defaultdict(Counter)
```

then start count directly

```python
    self._managed_tokens_cnt[ocm_env_name][dt_tenant_id] += 1
```

Reply:
Good idea! Fixed